### ISSUE TITLE
tensorflow.js open source

### ISSUE THREAD

**Issue by lupotto** [AIZOOTech/FaceMaskDetection#48]
Hello,

First off, great implementation. Thank you for sharing this. 

I am wondering if you are planning to provide the tensorflow.js implementation. I converted the model from keras to tensorflow.js, but It seems that the post-processing part should be done in order to print the bboxes and detections of the masks. 

Any idea or tip of how should be done?

Thank you in advance
